contrib/rpm/buildah.spec

```diff
@@ -26,7 +26,7 @@
 
 Name:           buildah
 # Bump version in buildah.go too
-Version:        1.20.0-dev
+Version:        1.20.1
```

Comment:
I think this should be 1.20.0

Reply:
This is the 1.20.1-dev patch branch and what setup.sh makes.

Reply:
nope, 1.20.0 is wrong too, looking

Reply:
1.20.1-dev
is what it should be

Reply:
I had that the last time, and the tests blew up.

Reply:
The tests may need to be adjusted too, it's probably looking at buildah.go still and needs to change to  define/types.go